slothy/targets/arm_v7m/arch_v7m.py

```diff
@@ -940,242 +966,242 @@ def make(cls, src):
 
 # movs
 class movw_imm(Armv7mBasicArithmetic): # pylint: disable=missing-docstring,invalid-name
-    pattern = "movw <Rd>, <imm>"
```

Comment:
Nit: I think one shouldn't be forced to remove those spaces -- it should be the parent class interpreting a space at this point as an optional space of arbitrary size?